source/js/buyers-guide/product-tab.js

```diff
@@ -0,0 +1,33 @@
+export default () => {
+  document.querySelectorAll(`#product-tab`).forEach((element) => {
+    const allTabs = element.querySelectorAll("span[data-product-label]");
+
+    // First tab is active by default
+    allTabs[0].classList.remove("tw-bg-white", "tw-text-black");
+    allTabs[0].classList.add("tw-bg-black", "tw-text-white");
+
+    allTabs.forEach((tabButton) => {
+      tabButton.addEventListener("click", (event) => {
+        event.stopPropagation();
+        event.preventDefault();
+        // reset active tab stylings
+        for (var i = 0; i < allTabs.length; i++) {
```

Comment:
let's use `allTabs.forEach` here as well - also note that `var` has been deprecated, use `let` for mutables, or `const` for immutables; both respect block scoping, which `var` doesn't do in the slightest =)